android-so/app/src/main/java/com/group/so/presentation/ui/customer/AddCustomerScreen.kt

```diff
@@ -113,7 +113,11 @@ fun AddScreenCustomer(
                 navigationIcon = {
                     IconButton(onClick = { navController.popBackStack() }) {
                         Icon(
-                            imageVector = Icons.Default.ArrowBack, contentDescription = "Back",
+                            // imageVector = Icons.Default.ArrowBack, contentDescription = "Back",
+                            painter = painterResource(id = R.drawable.ic_back),
+                            contentDescription = "Back",
```

Comment:
```suggestion
                            contentDescription = "Back",
```

This should received internacionalization?

Reply:
Text is not visible to the end user